sdk/monitor/Azure.Monitor.OpenTelemetry.Exporter/src/TelemetryPartB.cs

```diff
@@ -123,6 +129,44 @@ private static void AddPropertiesToTelemetry(IDictionary<string, string> destina
             }
         }
 
+        private static void AddActivityLinksToPartCTags(IEnumerable<ActivityLink> links, ref AzMonList PartCTags)
+        {
+            string msLinks = "_MS.links";
+            if (links != null && links.Any())
+            {
+                var linksJson = new StringBuilder();
+                linksJson.Append('[');
+                foreach (var link in links)
```

Comment:
Is there a limitation on how many links do we support? What do we do if we exceed the limit?

Reply:
That is a good point: I don't think we have a limit defined. I will double check.